includes/wc-formatting-functions.php

```diff
@@ -1108,7 +1122,7 @@ function wc_format_option_hold_stock_minutes( $value, $option, $raw_value ) {
  * @return string
  */
 function wc_sanitize_term_text_based( $term ) {
-	return trim( wp_unslash( strip_tags( $term ) ) );
+	return trim( wp_strip_all_tags( wp_unslash( $term ) ) );
```

Comment:
Curious why wp_strip_all_tags was added here since this function is not used as part of the coupon functionality?

Reply:
phpcs wanring for `strip_tags` usage.

Reply:
Gothca